### PR TITLE
Remove spm

### PR DESCRIPTION
**Pull request by faddat**
This PR will remove cosmoscmd/spm from Stargaze as it is known to cause issues with the cli and there have been 
reported cases of it causing instability.

@jhernandezb so, I think the right idea here is to have a look at this PR, note that there were some tests removed, and then we can discuss how to replace them.  This PR removes the testutil folder, and can be considered the basis of a clean upgrade to 47. 

This PR contains two other PR's:

* #792 
* #795 

Bascially, review and merge those and the changeset here will be reduced for easier review.... but also keep in mind that we should go over changes to tests before merging this.  I did simply comment some of them out, and this nukes the whole testutil folder.

### CODE REVIEW

**Comment by faddat** [comment]
Here's an example of a similar changeset, also in advance of a 47 upgrade

* https://github.com/cosmos/interchain-security/pull/812

**Comment by faddat** [comment]
Is it possible that this broke the test suite?

File: Makefile [hunk 1]
```diff
@@ -151,7 +151,7 @@ DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace $(
 ###############################################################################
 ###                                Protobuf                                 ###
 ###############################################################################
-PROTO_BUILDER_IMAGE=tendermintdev/sdk-proto-gen:v0.2
+PROTO_BUILDER_IMAGE=ghcr.io/cosmos/proto-builder:0.9.0
```
Comment (jhernandezb) [inline Makefile:154]: Question is there is a list of compatible container for proto-builder? I still struggle to find how this container is built!

**Comment by jhernandezb** [comment]
Taking a look at this now thanks for getting this started, spm has been annoying since day 1.

File: Makefile (hunk 1 above)
Comment (faddat) [reply Makefile:154]: here you go:

github.com/faddat/proto-builder

**Comment by faddat** [comment]
@jhernandezb / @spoo-bar -- I'm going to ask some of our team members if they can restructure the disabled tests.  

It is legitimately a lot of them.

**Comment by jhernandezb** [comment]
thank you @faddat will start looking on this for our v11 upgrade

**Comment by jhernandezb** [comment]
Closing spm was removed by a different PR

**Pull request closed without merging**